coordinator/src/orderbook/routes.rs

```diff
@@ -64,20 +67,27 @@ pub async fn post_order(
     let order = orderbook::db::orders::insert(&mut conn, new_order).unwrap();
 
     let sender = state.tx_pricefeed.clone();
-    update_pricefeed(PriceFeedMessage::NewOrder(order.clone()), sender);
+    update_pricefeed(PriceFeedResponseMsg::NewOrder(order.clone()), sender);
 
     Json(order)
 }
 
 #[derive(Serialize, Clone, Deserialize, Debug)]
-pub enum PriceFeedMessage {
+pub enum PriceFeedRequestMsg {
+    Authenticate(Signature),
+}
+
+#[derive(Serialize, Clone, Deserialize, Debug)]
+pub enum PriceFeedResponseMsg {
```

Comment:
The orderbook is all about a price feed?

Reply:
You are right, the orderbook is a price feed implicitly.

Even so, I think it would be clearer to use consistent terminology. `OrderbookResponse{Msg}` would be more straightforward given the name of the parent module.